functests/4_latency/latency.go

```diff
@@ -135,6 +135,9 @@ var _ = Describe("[performance] Latency Test", func() {
 			if isolatedCpus.Size() < 3 {
 				Skip(fmt.Sprintf("Skip the oslat test, the profile %q has less than two isolated CPUs", profile.Name))
 			}
+			if latencyTestCpus < 2 && latencyTestCpus != defaultTestCpus {
+				Skip("Skip the oslat test, LATENCY_TEST_CPUS is less than two CPUs")
```

Comment:
"is less than the minimum CPU amount or different from the default value %v", defeaultTestCpus

It is possible this check should split in two

Reply:
In fact, this is already covered when we parsed the cpus' number here :
[Here](https://github.com/openshift-kni/performance-addon-operators/blob/master/functests/4_latency/latency.go#L297 
)
So at the point we get to this condition, we have these options :
latencyTestCpus = default value (-1)
latencyTestCpus > -1 
& this condition will be true if the latencyTestCpus is 0 or 1 & only in this case we will skip the test.
However, I agree with changing "two CPUs" to "minimum allowed CPU amount for oslat tool which is 2"